spec/factories.rb

```diff
@@ -3,7 +3,7 @@
     candidate
     chased_id { candidate.id }
     chased_type { candidate.class }
-    chased_type { :referee_mailer_reference_request_chaser_email }
+    chased_type { :reference_request }
```

Comment:
This is a bit unclear to mee, you defined chased_type twice, was this a mistake?

Reply:
Yea it was meant to be chaser type. Good spot 👍